source/includes/_capture_external_event_api.md

```diff
@@ -0,0 +1,72 @@
+# Capture External Event API
+
+Capture External Event API is a public ReachLocal external interface for vendors to use.
+
+## Authorization - Capture External Event API
+
+All requests must include an [Authorization](#authentication) header.
+
+The email address used to obtain the OAuth access token must be associated with the GMAID of the requested advertiser.  If it is not, the API will return an HTTP response with status code 403 and the following body:
+
+`{ "message": "Forbidden" }`
+
+## Create Chat Event
+
+This endpoint is used to create a chat event.
+
+### HTTP Request
+
+| Method | URI Format |
+|---|---|
+| POST | https://externalapi.reachlocal.com/api/chats |
```

Comment:
The hostname of our API is https://api.reachlocalservices.com.  The path component for client_reports is /client_reports, then each client report endpoint is a sub-path under that.  I would suggest you use a path such as /capture_events with chat as a sub-path under that (https://api.reachlocalservices.com/capture_events/chat).

We will not be supporting externalapi.reachlocal.com in the future.

Reply:
Sounds good Brian. I took the URI from what we were using for thinking chat so I will change it per your recommendations.